src/Modal/Modal.js

```diff
@@ -98,8 +105,30 @@ function Modal({ children, title, animationDuration, showClose, onClose, open, .
     if (open !== isOpen) {
       setOpen(open);
     }
+
+    if (blur && blurNode) {
+      toggleBlur();
+    }
   }, [open]);
 
+  useEffect(() => {
+    if (blur) {
+      const target = document.getElementById(blurTarget);
+
+      if (target) {
+        setBlurNode(target);
+        target.style.transitionDuration = '750ms';
+        target.style.overflow = 'auto';
+
+        return () => {
+          target.style.transitionDuration = null;
+          target.style.filter = null;
+          target.style.overflow = null;
+        };
+      }
+    }
+  }, []);
```

Comment:
Do we need to add `[blur, blurTarget]` to the deps array since we want the effect to synchronize if those props change?

Reply:
We want to mimic a `componentDidMount` lifecycle method here. Which is why there is an empty deps array. If we add `[blur, blurTarget]` as the deps array, then it will keep grabbing applying those styles to the `#root` on every render. 

Those props shouldn't be changed frequently.

Reply:
>  If we add [blur, blurTarget] as the deps array, then it will keep grabbing applying those styles to the #root on every render.

Correct me if I'm wrong, but `#useEffect` will only reapply those styles ***if*** the deps array changes. If `blur` and `blurTarget` do not change between renders, `useEffect` will not re-run.

Reply:
You're 100% correct. Did some testing to verify and the cleanup still fires too. Changing to your suggestion :) :)